docs/Banner.md

```diff
@@ -0,0 +1,27 @@
+# `<Banner />`
+
+Use `<Banner />` component to create a box with a optional header, text, optional left-hand icon, optional right-hand close button, and palette color scheme.
+
+```jsx
+<Banner
+  palette='success'
+  header='Banner'
+  iconName='star'
+  iconSize={20}
+  text='This is a banner'
+  onClose={() => {}}
+  showIcon='false'
+  textAlign='right'
+/>
+```
+
+Prop | Type | Description
+---|---|---
+palette | string | Sets background-color and color. Accepts `default`, `information`, `success`, `attention`, `warning`, `gray`, `blue`, `green`, `orange`, `red`
```

Comment:
I think for the time being it would be better to keep the color props api similar across components and use either `color` or `bg` instead of `palette` – see the implementation of [Badge](https://github.com/pricelinelabs/design-system/blob/master/src/Badge.js) for a similar situation. ("Palette" also implies what `theme.colors` is in my mind)

Once we have a set of these more complex color combinations in a handful of components, I suspect we'll find a pattern that can be abstracted up to the theme level, if that makes sense